components/apimgt/org.wso2.carbon.apimgt.rest.api.authenticator/src/main/java/org/wso2/carbon/apimgt/rest/api/authenticator/AuthenticatorAPI.java

```diff
@@ -75,7 +75,7 @@
     @Path("/token/{appName}")
     public Response auth(){
       return Response.ok()
-                .header("Access-Control-Allow-Origin", "*")
+                .header("Access-Control-Allow-Origin", "https://localhost:9292")
```

Comment:
We should not hardcoded the hostnames here. Need to find a proper solution for this.

Reply:
@RenukaFernando Please note.

Reply:
Noted